app/src/main/java/org/fossasia/susi/ai/device/deviceconnect/DeviceConnectFragment.kt

```diff
@@ -45,6 +45,8 @@ class DeviceConnectFragment : Fragment(), IDeviceConnectView {
     private val VIEW_AVAILABLE_DEVICES = 1
     private val VIEW_AVAILABLE_WIFI = 0
     private var checkDevice: Boolean = false
+    val wifiIntentRequestCode = 1
+    val gpsIntentRequestCode = 0
```

Comment:
These are not constants, and probably the first or second lesson we learn in Java is to make variables private unless otherwise required